src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1089,6 +1092,13 @@ private void EnforceDoesNotReturn(SyntaxNode? syntaxOpt)
             Analyze(compilation, method, node, diagnostics, useConstructorExitWarnings, initialNullableState, getFinalNullableState, out finalNullableState);
         }
 
+#if DEBUG
```

Comment:
Part of the motivation for the feature flag was to disable in production but seems like you're making it debug only here.

Reply:
`CSharpCompilation.ShouldRunNullableAnalysis` now returns true when `run-nullable-analysis=always`.

---
In reply to: [539661298](https://github.com/dotnet/roslyn/pull/49876#discussion_r539661298) [](ancestors = 539661298)